lib/components/form/error-message.js

```diff
@@ -15,10 +22,10 @@ const ErrorMessage = ({ error, message }) => {
     <div className='error-message'>
       <div className='header'>
         <Icon type='exclamation-circle' />
-        <FormattedMessage id='components.ErrorMessage.header' />
+        <FormattedMessage id={warning ? 'components.ErrorMessage.warning' : 'components.ErrorMessage.header'} />
```

Comment:
Per a comment in another PR by @evansiroky, it is preferrable to issue a `<FormattedMessage id="..." />` for each message id rather than generate ids dynamically (this is so that `formatjs`' CLI tools can pick up the messages).

Reply:
Didn't realize it had to be the entire component rather than just the string! Adjusted in c47e62fa3758622229941f3636acbd33136ad89e